controllers/activegate/statefulset.go

```diff
@@ -195,10 +197,13 @@ func buildVolumeMounts(stsProperties *statefulSetProperties) []corev1.VolumeMoun
 }
 
 func buildEnvs(stsProperties *statefulSetProperties) []corev1.EnvVar {
```

Comment:
it's used onfly for `containers`, but not for `initContainers`. Open question - should we provide this env vars to `initContainers`? I think yes.

Reply:
There are no initContainers yet. Will be added by PR "Feature/k8s internal certificate import to the operator". So let's do it after the PR is merged.